pkg/apis/servicecatalog/v1beta1/types.go

```diff
@@ -427,6 +430,7 @@ type ExtraValue []string
 // In the future, this will be allowed and will represent the intention that
 // the ServiceInstance should have the plan and/or parameters updated at the
 // ClusterServiceBroker.
+// +k8s:openapi-gen=x-kubernetes-print-columns:custom-columns=NAME:.metadata.name,CLASS:.spec.clusterServiceClassExternalName,PLAN:.spec.clusterServicePlanExternalName,STATUS:.status.conditions[*].reason
```

Comment:
I wonder how flexible we can be here with status.  I'm not sure the current formulation of status is going to be useful - how about remove the `STATUS` bits and deal with status in a subsequent PR?